lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -17,24 +17,36 @@ bool IsCallOfConstants(ParseNode *pnode)
 template <class PrefixFn, class PostfixFn>
 void Visit(ParseNode *pnode, ByteCodeGenerator* byteCodeGenerator, PrefixFn prefix, PostfixFn postfix, ParseNode * pnodeParent = nullptr);
 
+//the only point of this type (as opposed to using a lambda) is to provide a named type in code coverage
```

Comment:
I think we can improve this comment to have a more positive feel. Instead of "the only point of" -- just say what it is for. "This type is used instead of a lambda to provide a named type in code coverage."